adagrams/game.py

```diff
@@ -1,11 +1,128 @@
+import random
+
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    list_letter_pool =[]
+    for letter, number in LETTER_POOL.items():
+        letter_quantity = letter * number
+        list_letter_pool.append(letter_quantity)
+
+
+    letter_string = ''.join(list_letter_pool)
+   
+    # Available letters with frequencies list
+    available_letters = list(letter_string)
+    
+    hand = []
+
+# Returns a list of 10 random letters
+    while len(hand) != 10:
+        random_letter = random.choice(available_letters)
+        hand.append(random_letter)
+        for letter in available_letters:
+            if letter in hand:
+                available_letters.remove(letter)
+    return hand
 
-def uses_available_letters(word, letter_bank):
-    pass
+    
+
+def uses_available_letters(word, hand):
+    # Changes user's input to uppercase
+    word = word.upper()  
+    # Returns true if all the letters in word are in the hand 
+    for letter in word:
+      if letter in hand and word.count(letter) < hand.count(letter):
+        continue
+      elif letter not in hand or word.count(letter) > hand.count(letter):
+        return False
```

Comment:
Good use of `continue`. In this case, I think our code still works if we get rid of the first `if` and only check the `elif` case (as an `if` case). For example, this still passes the `uses_available_letters` tests:

```python
    for letter in word:
      if letter not in hand or word.count(letter) > hand.count(letter):
        return False
```